testing/utils/iotest/burstbuffer-BP4-nsub1.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<adios-config>
+
+    <!--===========================================
+           Configuration for io_T1 group
+        ==========================================-->
+
+    <io name="io_T1">
```

Comment:
I'd try adding also YAML config files tests.

Reply:
habits just don't want to die...